src/java/org/apache/cassandra/db/lifecycle/LifecycleTransaction.java

```diff
@@ -142,7 +142,7 @@ public static LifecycleTransaction offline(OperationType operationType, SSTableR
     public static LifecycleTransaction offline(OperationType operationType, Iterable<SSTableReader> readers)
     {
         // if offline, for simplicity we just use a dummy tracker
-        Tracker dummy = new Tracker(null, false);
+        Tracker dummy = new Tracker(null, null, false);
```

Comment:
nit: `Tracker.NewDummyTracker()` or the like would be preferable:
- hides internal details (`null`, `null`, `false`) inside `Tracker`
- uses don't need to be changed the next time `Tracker` constructor changes